### ISSUE TITLE
Some errors because of focuses

### ISSUE THREAD

**Issue by Thanasis09** [dax0102/fuhrerredux-mod#19]
errors:
<details><summary>CLICK HERE</summary>
<p>

```lua
[16:25:58][nationalfocus.cpp:529]: Error in focus: fra_prov_govt (common/national_focus/FR_France.txt:613 )
relative_focus_id: fra_africa_missionary does not exist. Relative focus must be scripted before this.
[16:25:58][nationalfocus.cpp:529]: Error in focus: GER_rural_investsments (common/national_focus/FR_Germany.txt:14559 )
relative_focus_id: GER_overhaul_rural_infrastructures does not exist. Relative focus must be scripted before this.
[16:25:58][nationalfocus.cpp:529]: Error in focus: ARB_invite_kurdestan (common/national_focus/FR_Hashemite_Arabia.txt:1334 )
relative_focus_id: ARB_invite_persia does not exist. Relative focus must be scripted before this.
[16:25:58][nationalfocus.cpp:529]: Error in focus: ARB_snatch_the_trucial_states (common/national_focus/FR_Hashemite_Arabia.txt:1362 )
relative_focus_id: ARB_invite_persia does not exist. Relative focus must be scripted before this.
[16:25:58][nationalfocus.cpp:529]: Error in focus: NAR_forced_labour (common/national_focus/FR_New_American_Republic.txt:442 )
relative_focus_id: NAR_absorb_the_black_legion does not exist. Relative focus must be scripted before this.
[16:25:58][nationalfocus.cpp:529]: Error in focus: NAR_choose_the_entente (common/national_focus/FR_New_American_Republic.txt:1943 )
relative_focus_id: NAR_our_own_domain does not exist. Relative focus must be scripted before this.
[16:25:58][nationalfocus.cpp:529]: Error in focus: NAR_our_own_domain (common/national_focus/FR_New_American_Republic.txt:1972 )
relative_focus_id: NAR_join_the_entente does not exist. Relative focus must be scripted before this.
[16:25:58][database_scoped_variables.cpp:119]: invalid database object for effect/trigger:  file: common/national_focus/FR_Philippines.txt line: 4306 fighter_equipment. use var:var_name to explicitly use variables in effects/triggers
[16:25:58][nationalfocus.cpp:529]: Error in focus: ROM_green_agriarian_roots (common/national_focus/FR_Romania.txt:3748 )
relative_focus_id: ROM_their_assets_do_not_matter does not exist. Relative focus must be scripted before this.
[16:25:58][nationalfocus.cpp:529]: Error in focus: USA_lower_the_income_tax (common/national_focus/FR_United_States.txt:2476 )
relative_focus_id: USA_ensure_press_freedoms does not exist. Relative focus must be scripted before this.
[16:25:59][nationalfocus.cpp:497]: Error in focus: BLR_in_conserving_liberty (common/national_focus/FR_Belarus.txt:2430 )
Has mutual exclusivity with BLR_the_free_market_bears_fruits but the other focus does not.
```

</p>
</details>

**Comment by Thanasis09** [comment]
Closes #19
couldn't repro

**Comment by Thanasis09** [comment]
Reopens #19 because it reshowed me the errors.

**Comment by Thanasis09** [comment]
Fixed #38
